ospfd/ospf_vty.c

```diff
@@ -10707,56 +10742,9 @@ static void show_ip_ospf_route_network(struct vty *vty, struct ospf *ospf,
 						json_nexthop_array,
 						json_nexthop);
 				}
-				if (if_lookup_by_index(path->ifindex,
-						       ospf->vrf_id)) {
-
-					if (path->nexthop.s_addr
-					    == INADDR_ANY) {
-						if (json) {
-							json_object_string_add(
-								json_nexthop,
-								"ip", " ");
-							json_object_string_add(
-								json_nexthop,
-								"directly attached to",
```

Comment:
I'm afraid we might need to retain this field and mark it as deprecated before removing it.

Source: https://github.com/FRRouting/frr/issues/9197#issuecomment-887584753